packages/import-mistica-icons/index.js

```diff
@@ -1,4 +1,3 @@
-// @ts-check
```

Comment:
another option could be renaming this file to `.ts` and execute it with `ts-node`